client/my-sites/site-settings/controller.js

```diff
@@ -126,17 +130,17 @@ module.exports = {
 	},
 
 	deleteSite( context ) {
-		var site = sites.getSelectedSite();
+		let site = sites.getSelectedSite();
```

Comment:
Quick rule of thumb is to use `const` unless we actually reassign the reference. See also #4251